lib/linter.sh

```diff
@@ -865,7 +865,7 @@ LINTER_COMMANDS_ARRAY['BASH_EXEC']="bash-exec"
 LINTER_COMMANDS_ARRAY['CLANG_FORMAT']="clang-format --Werror --dry-run"
 LINTER_COMMANDS_ARRAY['CLOJURE']="clj-kondo --config ${CLOJURE_LINTER_RULES} --lint"
 LINTER_COMMANDS_ARRAY['CLOUDFORMATION']="cfn-lint --config-file ${CLOUDFORMATION_LINTER_RULES}"
-LINTER_COMMANDS_ARRAY['CLOUDFORMATION_CFN_NAG']="cfn_nag_scan --ignore-fatal --profile-path=${CLOUDFORMATION_CFN_NAG_LINTER_RULES} ${LINTER_OPTS[CLOUDFORMATION_CFN_NAG]}"
+LINTER_COMMANDS_ARRAY['CLOUDFORMATION_CFN_NAG']="cfn_nag_scan --ignore-fatal --blacklist-path=${CLOUDFORMATION_CFN_NAG_LINTER_RULES} ${LINTER_OPTS[CLOUDFORMATION_CFN_NAG]}"
```

Comment:
See https://github.com/stelligent/cfn_nag#global-blacklist

`--profile-path` specifies a list of rules that scans will _only_ check for
`--blacklist-path` specifies a global ignore list of rules (this is what we want for our use case)